server/utils/courseUtils.ts

```diff
@@ -34,9 +34,13 @@ const prerequisiteSummaryListRows = (prerequisites: Array<CoursePrerequisite>):
   prerequisites.forEach(prerequisite => {
     const index = order[prerequisite.name]
 
-    summaryListRows[index] = {
-      key: { text: prerequisite.name },
-      value: { text: prerequisite.description },
+    if (summaryListRows[index]) {
```

Comment:
Let's use `continue` above here rather than the `reduce` and disable ESLint's complaints

Reply:
(`return` was what I should have said...)